dataland-frontend/tests/component/utils/TypeScriptUtils.cy.ts

```diff
@@ -15,4 +15,9 @@ describe('Unit test for assertDefined', () => {
     expect(assertDefined('Test')).to.equal('Test');
     expect(assertDefined(5)).to.equal(5);
   });
+
+  it('verifies that the isString function works', () => {
+    expect(isString('Test')).to.be.true;
+    expect(isString(5)).to.be.false;
```

Comment:
magic numer could be defined at the top of the test?

Reply:
Fixed